### PR TITLE
adding in optional SCORECARD_WAIT_TIME env variable to pass to scorecard

### PR DESCRIPTION
**Pull request by acornett21**
- Fixes: #367 
- Adding in SCORECARD_WAIT_TIME so that if user's experience a timeout and preflight returns an error saying they can try to increase the timeout, this actually exposes that value so they can try again with an increased value before they raise an issue against preflight.
- Updated documentation

Signed-off-by: Adam D. Cornett <adc@redhat.com>

### CODE REVIEW

**Review by samira-barouti — APPROVED** [review]
/lgtm

File: certification/internal/policy/operator/scorecard_check.go [hunk 1]
```diff
@@ -42,7 +45,7 @@ func (p *scorecardCheck) getDataToValidate(bundleImage string, selector []string
 		Namespace:      namespace,
 		ServiceAccount: serviceAccount,
 		Verbose:        true,
-		WaitTime:       "240s",
+		WaitTime:       fmt.Sprintf("%s%s", waitTime, "s"),
```
Comment (bcrochet) [inline certification/internal/policy/operator/scorecard_check.go:48]: Small nit, feel free to ignore, but... The second arg? Is that necessary?

**Review by bcrochet — COMMENTED** [review]
Small nit, but otherwise lgtm

File: certification/internal/policy/operator/scorecard_check.go (hunk 1 above)
Comment (acornett21) [reply certification/internal/policy/operator/scorecard_check.go:48]: @bcrochet You're thinking`fmt.Sprintf("%ss", waitTime)` right? That's probably clearer/cleaner.

Comment (acornett21) [reply certification/internal/policy/operator/scorecard_check.go]: Updated this for clarity.

**Review by bcrochet — APPROVED** [review]
/lgtm

**Review by jomkz — APPROVED** [review]

**Pull request merged**